src/Microsoft.DotNet.Wpf/src/Shared/MS/Internal/IO/Packaging/PackagePartExtensions.cs

```diff
@@ -0,0 +1,85 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using MS.Internal.WindowsBase;
+using System.IO;
+using System.IO.Packaging;
+
+namespace MS.Internal.IO.Packaging
+{
+    /// <summary>
+    /// Extensions to provide wrappers for functionality that no longer exists in System.IO.Packaging.PackagePart
+    /// </summary>
+    [FriendAccessAllowed]
```

Comment:
Is this only used by WindowsBase? If so, does this need to be `FriendAccessAllowed`?

Reply:
It's included by base, used by core and framework.  I wasn't going to add it to shared, but the other utility file is already there (also only included by base and used by core and framework as well) so I figured put it there for now.